python/PyQt6/analysis/auto_generated/raster/qgsninecellfilter.sip.in

```diff
@@ -52,6 +52,24 @@ Starts the calculation, reads from mInputFile and stores the result in mOutputFi
     double outputNodataValue() const;
     void setOutputNodataValue( double value );
 
+    void setCreateOptions( const QStringList &list );
```

Comment:
```suggestion
    void setCreationOptions( const QStringList &list );
```

Reply:
I used the same method name as in `QgsRasterFileWriter`. But I agree that "creation options" is more in line with GDAL. Should I rename similar methods in other places, keeping backward compatibility?

Reply:
Yes please, if it doesn't break the API.